packages/orchestration/src/orchestration-utils.test.ts

```diff
@@ -82,7 +83,10 @@ describe('orchestration utils', () => {
       const config: OrchestrationModuleConfig = {
         ...defaultOrchestrationModuleConfig,
         filtering: {
-          output: buildAzureContentFilter({ Hate: 4, SelfHarm: 0 })
+          output: buildAzureContentFilter({
+            Hate: AzureFilterThreshold.ALLOW_SAFE_LOW_MEDIUM,
+            SelfHarm: AzureFilterThreshold.ALLOW_SAFE
+          })
```

Comment:
[q] Can we avoid changing these tests and only add new tests (even just type tests) to avoid merge conflicts with #441? I think it is unnecessary to change those types.